netlify.toml

```
@@ -23,7 +23,7 @@
 
 [[redirects]]
   from = "/scipylib/dev-zone.html"
-  to = "/dev-zone/"
+  to = "/community/"
```

I think this should be
```suggestion
  to = "/contribute/"
```